### ISSUE TITLE
Unable to create service control instances when upgrading to 4.26.0

### ISSUE THREAD

**Issue by TraGicCode** [Particular/ServiceControl#3200]
Once you upgrade to v4.26.0 of servicecontrol new instance creation no longer works.  Below is a screenshot of the error which doesn't really provide much information.

![image](https://user-images.githubusercontent.com/6224270/201481101-359be059-de79-4f0d-ad6c-a0c82a02792f.png)

Reproduction Steps
-------------------------
1. Install servicecontrol v4.25.2
2. Create an error and audit instance
3. Upgrade servicecontrol to v4.26.0
4. Attempt to create a new audit/error instance and notice it always fails resulting in the screenshot i posted above

**NOTES: I assume monitoring instance is the same but did not check it**

**Comment by mauroservienti** [comment]
@TraGicCode, sorry for the issue. We tried multiple upgrade scenarios with no issues. It's clear that there are more. Can you share the tooltip text, if any, that should be displayed when hovering with the mouse on the error message?

**Comment by mauroservienti** [comment]
I can reproduce the issue. In my case, the errors were:

- the primary instance port was already in use by another instance
- the primary instance database maintenance port was already in use by another instance
- the same as above for the audit instance
- Given that, the second time I tried installing again using MSMQ as the transport, the error queue was already assigned to the first ServiceControl

As a workaround, I manually adjusted the ports and changed the error queue name. Are the above the same errors you're facing?
